testing/razercfg: new aport

tanertas opened the pull request:
https://bues.ch/cms/hacking/razercfg.html
Razer mouse configuration tool.

tanertas commented:
@andypost can you take a look at this?

clandmeter commented on testing/razercfg/APKBUILD:
```diff
@@ -0,0 +1,42 @@
+
+build() {
+	cd "$builddir"
+	cmake . -DCMAKE_INSTALL_PREFIX='/usr'
```
You are not running make here?
I think make install is not triggering it which is not the correct location.

tanertas replied:
I'm not sure what is your confusion bu I have to say that his is the cmake equivalent of --prefix=/usr.

maxwell-k replied:
Maybe I can explain a little more:

The README says that both cmake and make are used for the build step. cmake creates the Makefile. make uses the Makefile to build the binaries. https://github.com/mbuesch/razer#building

make install is correctly called in the package step. If the binary files are not available, so if make has not been called in the build step, make will notice and compile them. Compilation should not happen in the package step, so we're suggesting you add a make call to build().

clandmeter commented:
Please also rebase against master so CI picks it up.

tanertas commented:
> Please also rebase against master so CI picks it up.

Why rebase? There's no conflict with master anyway since this is a new aport PR. CI already triggered when I update PR anytime.
Btw, I updated initscript.

tanertas commented:
> > Please also rebase against master so CI picks it up.
> 
> Why rebase? There's no conflict with master anyway since this is a new aport PR. CI already triggered when I update PR anytime.
> Btw, I updated initscript.

@clandmeter I realized that you were referring to the "drone" CI specifically. Sorry for misunderstood. I rebased to master.

Ikke commented:
Is this significant?

> WARNING: ldconfig failed: 1
Your system will probably be unable to locate librazer.so library
